### PR TITLE
Fix typo in Input.hs

### LINKED ISSUE #836: rename `Input.FindShallow` to `Input.FindShallowI`
in `src/Unison/Codebase/Editor/Input.hs`

### PR DESCRIPTION
**Pull request by SunriseM**
Fixes #836

### CODE REVIEW

**Comment by aryairani** [comment]
This is great, thanks!  Add yourself to contributors.md file, and then we’ll get this merged!

File: parser-typechecker/src/Unison/Codebase/Editor/Input.hs [hunk 1]
```diff
@@ -103,7 +103,7 @@ data Input
   | LinksI Path.HQSplit' (Maybe String)
   -- other
   | SearchByNameI Bool Bool [String] -- SearchByName isVerbose showAll query
-  | FindShallow Path'
+  | FindShallowl Path'
```
Comment (TomasMikula) [inline parser-typechecker/src/Unison/Codebase/Editor/Input.hs]: I think this should be `FindShallowI`, i.e. the last letter is `I` (uppercase `i`), not lowercase `L`.

**Comment by SunriseM** [comment]
@TomasMikula Thanks for the correction. Fixed

File: parser-typechecker/src/Unison/Codebase/Editor/Input.hs (hunk 1 above)
Comment (aryairani) [reply parser-typechecker/src/Unison/Codebase/Editor/Input.hs]: @TomasMikula Thanks for catching this.

**Comment by pchiusano** [comment]
Thanks again!

**Pull request merged**